dev/tools/controllerbuilder/pkg/codegen/common.go

```diff
@@ -56,7 +56,7 @@ var Acronyms = []string{
 	"MiB",
 	"NAT",
 	"OAuth2", "OIDC", "OS",
-	"PD", "PSC",
+	"PD", "PSC", "PZI", "PZS",
```

Comment:
ditto, remove PZI and PZS since we drop those two fields and they are not well known abbreviations.

Reply:
Removed.

Though I think whether a word is an abbreviation is objective (not subjective) regardless whether people know it. Using `Pzi`/`Pzs` will probably be more confusing then marking them as `PZI`/`PZS` directly.